src/agent/samples/policy/outdated/ctr/ctr-busybox-data.json

```diff
@@ -0,0 +1,208 @@
+{
```

Comment:
If you want to reduce the diff, you can delete all the files under outdated/. Those files haven't been functional for a several months now.